test/alphabet/gap/gapped_alphabet_test.cpp

```diff
@@ -43,74 +41,186 @@
 
 using namespace seqan3;
 
-TEST(gapped_alphabet_test, test_alphabet_concept)
+TEST(gapped_alphabet_test, default_constructor)
 {
-    EXPECT_TRUE(alphabet_concept<gapped_alphabet<dna4>>);
+    using alphabet_t = gapped_alphabet<dna4>;
+
+    constexpr auto letter1 = alphabet_t{};
+    EXPECT_EQ(letter1._value, 0);
 }
 
-TEST(gapped_alphabet_test, test_implicit_inner_type_compatibility)
+TEST(gapped_alphabet_test, base_alphabet_copy_constructor)
```

Comment:
it is the copy constructor syntax, but using a base alphabet instance. I appreciate every better wording.